ckan/lib/helpers.py

```diff
@@ -45,6 +45,7 @@
 import ckan.plugins as p
 import ckan
 
+from ckan.lib.pagination import Page
```

Comment:
I think we should actually drop ckanext-qgov's dependency on this, but that can happen later.